packages/terra-framework-docs/src/terra-dev-site/test/pills/Pills/RemovablePill.test.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import Pills from '@cerner/terra-pills/lib/index';
+
+const BasicRemovablePill = () => {
+  const pills = [
+    {
+      label: 'asthma',
+      id: 'terra-removable-pill-asthma',
+      isRemovable: false,
+    },
+    {
+      label: 'bronchitis',
+      id: 'terra-removable-pill-bronchitis',
+      isRemovable: false,
+    },
+    {
+      label: 'fibro',
+      id: 'terra-removable-pill-fibro',
+      isRemovable: true,
+    },
+    {
+      label: 'asthma',
+      id: 'terra-removable-pill-asthma1',
+      isRemovable: false,
+    },
+    {
+      label: 'bronchitis',
+      id: 'terra-removable-pill-bronchitis1',
+      isRemovable: true,
+    },
+    {
+      label: 'fibro',
+      id: 'terra-removable-pill-fibro1',
+      isRemovable: false,
+    },
+  ];
```

Comment:
can we move this pills data to separate file and loop it through `React.Children.map` in the required test files.

Reply:
The pill data is not identical in most scenarios. If there arises such a situation, I shall move it to a single file.